src/builtins/register.rs

```diff
@@ -1,5 +1,4 @@
 //! Utilities for registering application commands
-
 use crate::serenity_prelude as serenity;
```

Comment:
Tiny thing but could you add back the newline :innocent: All files have space between the module docs and the other items, and I think it looks neater (I personally separate each group of items at the top of a file with a newline)